spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -19,8 +19,8 @@
     end
 
     it 'calls check_existence_for_dir with the right arguments' do
-      HostSettings.storage_roots.each do |storage_root|
-        expect(described_class).to receive(:check_existence_for_dir).with("#{storage_root[1]}/#{Settings.moab.storage_trunk}")
+      HostSettings.storage_roots.to_h.each_value do |path|
```

Comment:
i'd rather we went with `storage_root_location` rather than `path`, mimicing the other tests changed.

Reply:
Because it is a path, and that is sufficient for understanding the test on the *very next line*.  We do pathy things with it.  The value in question was not even named previously, it was just `storage_root[1]`.  This feels like an incredibly random nit after a month of time and multiple rebases.